styles/native/ts/core/widgets/floatingactionbutton.ts

```diff
@@ -67,6 +68,8 @@ export const com_mendix_widget_native_floatingactionbutton_FloatingActionButton:
     secondaryButtonCaptionContainer: {
         // All ViewStyle properties are allowed
         marginHorizontal: 5,
+        backgroundColor: background.secondary,
+        borderColor: darkMode ? background.secondary : "#eee",
```

Comment:
I think it makes more sense to just use `background.secondary` for the border as well, regardless of the theme. So if someone changes `background.secondary` they won't run into the same problem because the styles will automatically adapt.

Reply:
I see what you mean. A consideration is backwards compatibility: `#eee` has been the borderColor for awhile, changing it to `background.secondary` is a sudden change, although the two colors are similar.
![Screenshot 2021-12-10 at 09 15 53](https://user-images.githubusercontent.com/6040547/145540566-9fcf6f56-da46-4adc-a50c-1b713ed44136.png)

Reply:
@wesleysieses Changed